libs/orchestrator-ui-components/src/lib/components/OrchestratorPageTemplate/OrchestratorPageHeader.tsx

```diff
@@ -74,22 +81,17 @@ export const OrchestratorPageHeader: FC<OrchestratorPageHeaderProps> = ({
                     />
                 </EuiHeaderSectionItem>
                 <EuiHeaderSectionItem>
-                    <HeaderBadge color="warning">Development</HeaderBadge>
+                    <EnvironmentBadge />
                 </EuiHeaderSectionItem>
             </EuiHeaderSection>
 
             <EuiHeaderSection>
                 <EuiHeaderSectionItem>
                     <EuiBadgeGroup css={{ marginRight: multiplyByBaseUnit(2) }}>
-                        <HeaderBadge
-                            color="emptyShade"
-                            iconType={() => (
-                                <StatusDotIcon color={theme.colors.success} />
-                            )}
-                        >
-                            Engine is {engineStatus?.global_status}
-                        </HeaderBadge>
-                        <FailedTasksBadge {...taskCountsSummary} />
+                        <EngineStatusBadge engineStatus={engineStatus} />
```

Comment:
Since you made it specific for engineStatus, use `useEngineStatusQuery` inside the component instead of passing it from here.